src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/HaFlowAllEntityPaths.groovy

```diff
@@ -0,0 +1,56 @@
+package org.openkilda.functionaltests.helpers.model
+
+import org.openkilda.functionaltests.helpers.model.FlowPathModel.DiverseGroupModel
+import org.openkilda.northbound.dto.v2.haflows.HaFlowPaths
+import org.openkilda.testing.model.topology.TopologyDefinition
+
+import groovy.transform.EqualsAndHashCode
+import groovy.transform.ToString
+import groovy.transform.builder.Builder
+
+@EqualsAndHashCode(excludes = "topologyDefinition")
+@Builder
+@ToString(includeNames = true, excludes = 'topologyDefinition', includePackage = false)
+class HaFlowAllEntityPaths {
+
+    FlowPathModel shared
+    List<FlowPathModel> subFlowPaths
```

Comment:
Let's be consistent. Either `sharedPaths` or `subFlows`.

Reply:
Done.